src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java

```diff
@@ -164,26 +166,40 @@ private Uni<AsyncResultSet> executeSchemaChange(SimpleStatement boundStatement,
                         .setIdempotent(true)
                         .setSerialConsistencyLevel(
                             operationsConfig.queriesConfig().consistency().schemaChanges())))
-        .onFailure(DriverTimeoutException.class)
+        .onFailure(
+            error ->
+                error instanceof DriverTimeoutException || error instanceof InvalidQueryException)
         .recoverWithUni(
             throwable -> {
-              logger.error("Timeout executing schema change query : {}", boundStatement.getQuery());
+              logger.error(
+                  "Timeout/Invalid query executing schema change query : {}",
+                  boundStatement.getQuery());
               SimpleStatement duplicate = SimpleStatement.newInstance(boundStatement.getQuery());
               return Uni.createFrom()
-                  .completionStage(
-                      cqlSessionCache
-                          .getSession()
-                          .executeAsync(
-                              duplicate
-                                  .setExecutionProfileName(profile)
-                                  .setIdempotent(true)
-                                  .setSerialConsistencyLevel(
-                                      operationsConfig
-                                          .queriesConfig()
-                                          .consistency()
-                                          .schemaChanges())));
+                  .item(throwable)
+                  .onItem()
+                  .delayIt()
+                  .by(Duration.ofSeconds(1))
```

Comment:
Would it make sense to make this configurable? (and if so, specify in milliseconds, not seconds, to allow things like `200` msecs)

Reply:
Made it configurable. But from what we saw this needs delay in seconds.

Reply:
I am not sure I follow... why in seconds? I mean, default value being 1 second is probably fine and makes sense, but is it never useful to configure it in fractions (1.5 seconds etc)?

Reply:
Changed to millis as suggested.